File: pkg/models/Measurement.go
package models

import "time"

type Measurement struct {
	Id          int64     `json:"id" gorm:"type:serial;primary_key"`
	Date        time.Time `json:"date" gorm:"type:timestamp"`
	Temperature float64   `json:"temperature"`
	Humidity    float64   `json:"humidity"`
	Voltage     float64   `json:"voltage"`
	Signal      float64   `json:"signal"`
	TagUUID     string    `json:"tag_uuid" gorm:"type:uuid;not null"`
}

func (Measurement) TableName() string {
	return "measurement"
}

File: fetch_service/src/main/java/ru/tagmeasurements/fetch_service/models/Measurement.java
package ru.tagmeasurements.fetch_service.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.time.LocalDateTime;
import java.util.UUID;

@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
public class Measurement {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private LocalDateTime date;
    private Double temperature;
    private Double humidity;
    private Double voltage;
    private Double signal;
    @Column(name = "tag_uuid")
    private UUID tagUUID;
}
